build.sbt

```diff
@@ -98,8 +102,43 @@ lazy val junitDriver = module("junit-driver", "driver/bindings/junit")
     libraryDependencies ++= Dependencies.junit
   )
 
+lazy val scalaProbeApi = project(id = "scala-probe-api", path = "extensions/scala/api", publish = true).dependsOn(api)
+
+lazy val scalaProbePlugin =
+  ideaPluginModule(id = "scala-probe-plugin", path = "extensions/scala/probePlugin", publish = true)
+    .dependsOn(probePlugin, scalaProbeApi)
+    .settings(
+      intellijPluginName := "ideprobe-scala",
+      crossScalaVersions := List(scala212),
```

Comment:
I think we probably need to only publish 2.12 for all of scala-probe modules. It will be difficult to run driver on 2.13 and internals on 2.12